lib/improver/cube_combiner.py

```diff
@@ -205,16 +124,21 @@ def process(self, cube_list, new_diagnostic_name,
                 Revised coordinates for combined cube.
             revised_attributes (dict or None):
                 Revised attributes for combined cube.
-
+            expanded_coord (dict or None):
+                Coordinates to be expanded as a key, with the value
+                indicating whether the upper or mid point of the coordinate
+                should be used as the point value, e.g.
+                {'time': 'upper'}.
```

Comment:
Note this doc-string was simply missing, this is not actually a change to functionality.